Sources/Turf/Consumer/Consumer+WKT.swift

```diff
@@ -0,0 +1,327 @@
+            ),
+        ])
+    ])
+    
+    /*
+     GEOMETRYCOLLECTION (POINT (40 10),
+     LINESTRING (10 10, 20 20, 10 40),
+     POLYGON ((40 40, 20 45, 45 30, 40 40)))
+     */
+    private let geometryCollection: Consumer<WKTLabel> = .label(.geometryCollection, [
+        .spaced(.string("GEOMETRYCOLLECTION")),
+        .any([
+            .reference(.empty),
+            .bracketed(
+                .interleaved(
+                    .spaced(.reference(.object)),
+                    .discard(",")
+                )
+            )
+        ])
+    ])
+    
+    private lazy var object: Consumer<WKTLabel> = {
+        let primitives = empty | number | coordinate | coordinatesArray | coordinatesArray2D | coordinatesArray3D | space
+        let geometries = geometryCollection | point | multiPoint | lineString | multiLineString | polygon | multiPolygon
+        return .label(.object, [
+            primitives | geometries
+        ])
+    } ()
+    
+    private lazy var wktConsumer: Consumer<WKTLabel> = .interleaved([
+        .spaced(object)
+    ], .discard(","))
+    
+    // Transform
+    private let wktTransform: Consumer<WKTLabel>.Transform = { label, values in
+        switch label {
+        case .empty:
+            return nil
+        case .number:
+            guard let numbers = (values as? [String])?.compactMap({ Double($0) }),
+                  numbers.count == values.count,
+                  let number = numbers.first else {
+                throw WKTError.numberParsingFailed(values)
+            }
+            return number
+        case .coordinate:
+            guard let coords = values as? [Double],
+                  coords.count == 2 else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return LocationCoordinate2D(latitude: coords[1],
+                                        longitude: coords[0])
+        case .coordinatesArray:
+            guard let coords = values as? [LocationCoordinate2D] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return coords
+        case .coordinatesArray2D:
+            guard let coords = values as? [[LocationCoordinate2D]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return coords
+        case .coordinatesArray3D:
+            guard let coords = values as? [[[LocationCoordinate2D]]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return coords
+        case .point:
+            guard values.count > 1 else { return nil }
+            guard let coords = values[1] as? LocationCoordinate2D else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return Point(coords)
+        case .multiPoint:
+            guard values.count > 1 else { return nil }
+            let coords = values[1..<values.endIndex].compactMap { $0 as? LocationCoordinate2D }
```

Comment:
This is correct, but we can avoid a range here and in similar situations for additional safety assurance:

```suggestion
            let coords = values.suffix(from: 1).compactMap { $0 as? LocationCoordinate2D }
```